pkg/utils/resources.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+// Inspired from SNR - https://github.com/medik8s/self-node-remediation/blob/main/controllers/selfnoderemediation_controller.go#L283-L346
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	loggerResouces = ctrl.Log.WithName("resource")
```

Comment:
typo s/loggerResouces/loggerResources/

Reply:
I would call it just `logger` or even `log`. Why do we need the package name in the name?

Reply:
> Why do we need the package name in the name?

Just for easier identification of the log's source.

Reply:
I mean the var name :)